docs/web-component.md

```diff
@@ -0,0 +1,896 @@
+```javascript
+const template = document.createElement('template');
+template.innerHTML = `
+<!DOCTYPE html>
+<html lang="EN">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">
+    <style>     </style>
+    <script></script>
+</head>
+<body>
+      ......
+      <main class="fd-page">.....</main>
+      ......
+</body>
+`;
+
+export default class extends HTMLElement {
+  constructor() {
+     super();
+     this.attachShadow({ mode: 'open' });
+     this.shadowRoot.appendChild(template.content.cloneNode(true));
+     ..........
+  }
+}
+```
+              
+
+## Tip: Loading a Web Component inside a Luigi WebComponent
+In order to modularize/reuse some code, you would like to import other standard Web Component inside Luigi Web Component. 
+There could also be the case that you have already micro front end running in a standard Web Component and you would like to reuse it: 
+the easier way is to create a Luigi Proxy Web Component, here below and example:
+
+```javascript
+//In Luigi Core Config you will have this new Item
+{ 
+    pathSegment: 'your-web-componet-through-proxy', 
+    ………    
+    viewUrl: '/wc/luigi_proxy_wc.js',
+    webcomponent: true,
+    ………    
+}
+
+```
+
+```javascript
+//This would be the file luigi_proxy_wc.js
+
+const template = document.createElement('template');
+template.innerHTML = `
+  
+  <your-web-component option1='xxx' option2='yyyyyy'/>
+     
+`;
+
+import "./your_custom_wc.js";
+
+export default class extends HTMLElement {
+  constructor() {
+     super();
+     this.attachShadow({ mode: 'open' });
+     this.shadowRoot.appendChild(template.content.cloneNode(true));
+     ..........
+  }
+}
+```
+
+```javascript
+//This would be the file your_custom_wc.js
+
+(function() {
+  class F_Input extends HTMLElement {
+    constructor() {
+      super();
+      this.shadow = this.attachShadow({ mode: 'open' });
+      .........
+   }
+  }
+  
+  customElements.define('your-web-component', F_Input);
+
+})();
+```
+
+
+## Tip: Example of Luigi Web Component using other standard Web Component and how to communicate with them.
+
+You can have a look at our Luigi Fiddle example where we showcase a Luigi Web Component that is using other custom standard web components and Luigi Client object [here](https://fiddle.luigi-project.io/#/home/wc2)
+The following section gives a short description of all the files that are used.
+
+You can see how the different Web Components are injected through the diagrams below:
+
+![Component Injection](assets/component_injections.png)
+
+The Components cannot use (read, write) other Components elements. For this reason, we created a new simple event mechanism example to allow communication between the different components:
+
+![Component Events](assets/component_events.png)
```

Comment:
Having a random look: I assumed this would not show in [normal GitHub Markdown Viewer](https://github.com/SAP/luigi/blob/59e348fc34394834d8722fe4d6e9cf2e24c2e483/docs/web-component.md). 
But I also can't find these PNG Image files in this PR. Probably wasn't committed, or do we even have special images 🤔  ?

Reply:
It would be easier probably if it is added as an external link, like in the image of  [Getting Started](https://github.com/ndricimrr/luigi/blob/master/docs/getting-started.md):
`<img src="https://github.com/SAP/luigi/blob/master/docs/assets/luigi-overview-diagram.jpg?raw=true" width="600"/>`
Example:
<img src="https://github.com/SAP/luigi/blob/master/docs/assets/luigi-overview-diagram.jpg?raw=true" width="600"/>

Reply:
I just added the images....